validator/testcases/installrdf.py

```diff
@@ -53,7 +53,6 @@ def _test_rdf(err, install):
                       "updateInfoURL",
                       "updateKey",
                       "updateURL",
-                      "updateLink",  # Banned, but if not, pass it once.
```

Comment:
It's never actually existed. It's an element in update.rdf, but not in install.rdf. Donno why it was there.